system/src/system_power_manager.cpp

```diff
@@ -41,6 +41,8 @@ constexpr system_tick_t DEFAULT_FAULT_SUPPRESSION_PERIOD = 60000;
 
 constexpr system_tick_t DEFAULT_QUEUE_WAIT = 1000;
 constexpr system_tick_t DEFAULT_WATCHDOG_TIMEOUT = 60000;
+// FIXME: make sure to change setWatchdog() call to use the same interval
```

Comment:
When will we make this setWatchdog() change?

Reply:
![image](https://user-images.githubusercontent.com/388478/96504035-58cde580-127e-11eb-8877-c45c2a379233.png)